common/rya.api/pom.xml

```diff
@@ -95,5 +95,20 @@ under the License.
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <!-- Trivial listing of classes to be loaded via 
+                            SPI -->
+                        <exclude>src/main/resources/META-INF/services/org.openrdf.query.algebra.evaluation.function.Function</exclude>
```

Comment:
Remove this.  Add the license to the service loader.  # is the comment character
http://docs.oracle.com/javase/7/docs/api/java/util/ServiceLoader.html